source/anti-patterns/language.md

```diff
@@ -70,23 +74,9 @@ with (import <nixpkgs> {});
 # try this instead:
 let
   pkgs = import <nixpkgs> {};
-  inherit (pkgs) curl jq;
 in ...
 ```
 
-```nix
```

Comment:
Since I can't make suggestions on deleted lines (WTF GitHub?) and lines not modified:

l. 68:

> We recommend the following:

l. 80:

> Use `with` only for tight scopes which leave no ambiguity:
> 
> ```
> buildInputs = with pkgs; [ curl jq ];
> ```
> ```
> meta.maintainers = with lib.maintainers; [ edolstra ];
> ```